distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -1416,6 +1493,220 @@ type Builder
            changes to the builder will not affect the returned vector.
         Vector.from_polyglot_array self.java_builder.toArray
 
+## PRIVATE
+type Build_Style_Builder
```

Comment:
Reason for a new Builder:

Because the new builder calls `to_vector` for you, it can be difficult to ensure that warnings attached in the builder callback are propagated. (This is because these are only propagated if they occur in the same code that calls `to_vector`, so that the warnings are propagated through the `to_vector` call.) I had originally modified the existing `Builder` to take a parameter altering its behavior for `Vector.build`, but decided to make it a separate builder type, since we are hoping to remove the `new_builder` pattern entirely.